endorser/endorser_processor.py

```diff
@@ -305,13 +305,9 @@ async def is_valid_issuer(did: str, schema_id: str):
         )
         return False
 
-    schemas = schema_res.json()["schemas"]
-    if schema_id not in schemas:
-        logger.info(
-            "Schema `{}` associated with endorsement request is not in the trust registry.",
-            schema_id,
-        )
-        return False
+    schema = schema_res.json()
+    if not schema:
+        bound_logger.error("Schema {} not found in trust registry", schema_id)
```

Comment:
If the try succeeds can't we just move on to end of function and return True? Because at that point we know our schema is there and we dont need to schema = schema_res.json()

Reply:
Agreed - in this case of just checking if the schema exists in the db, a 200 response would indicate that already